apps/src/craft/craft.js

```diff
@@ -305,26 +307,29 @@ Craft.init = function (config) {
     }
   };
 
-  var codeWorkspace = (
-    <CodeWorkspace
-      localeDirection={studioApp.localeDirection()}
-      editCode={!!config.level.editCode}
-      readonlyWorkspace={!!config.readonlyWorkspace}
-      isMinecraft={true}
-    />
+  // Push initial level properties into the Redux store
+  studioApp.reduxStore.dispatch(setInitialLevelProps({
+    localeDirection: studioApp.localeDirection(),
+    isReadOnlyWorkspace: !!config.readonlyWorkspace,
+    isDroplet: !!config.level.editCode
+  }));
+
+  ReactDOM.render(
+    <Provider store={studioApp.reduxStore}>
+      <AppView
+          assetUrl={studioApp.assetUrl}
+          isEmbedView={!!config.embed}
+          isShareView={!!config.share}
+          hideSource={!!config.hideSource}
+          noVisualization={false}
+          isRtl={studioApp.isRtl()}
+          codeWorkspace={<ConnectedCodeWorkspace isMinecraft={true}/>}
```

Comment:
Yeah, I'd like to move CodeWorkspace into the AppView. I was kind of planning on doing that all in a separate PR though. At that point the "other way" of handling `isMinecraft` is presumably to stick it into the store (either as an initialLevelProperty or by whatever equivalent we use for stuff local editCode)